third-party/realdds/src/topics/image-msg.cpp

```diff
@@ -15,13 +18,15 @@ namespace topics {
 namespace device {
 
 
-image::image( const raw::device::image & image )
-    : raw_data( image.raw_data() )  // TODO: avoid image copy?
-    , width( image.width())
-    , height( image.height() )
-    , size( image.size() )
-    , format( image.format() )
+image & image::operator=( raw::device::image && rhs )
 {
+    raw_data = rhs.raw_data();
```

Comment:
See my other comment (so this should go away), but still: if `rhs` is `&&` then this should be a `move`.